### PR TITLE
feat: allow the configuration of elasticsearch ssl

### PR DESCRIPTION
**Pull request by JulienFauvel**
Closes gravitee-io/issues#3204

### CODE REVIEW

File: apim/README.md [hunk 1]
```diff
@@ -127,14 +127,20 @@ See [MongoDB replicaset](https://github.com/helm/charts/tree/master/stable/mongo
...
+| `es.tls.enabled`           | Elasticsearch TLS enabled                         | false                                                                  |
+| `es.tls.keystore.type`     | Elasticsearch TLS keystore type (jks, pem or pfx) | `null`                                                                 |
+| `es.tls.keystore.path`     | Elasticsearch TLS keystore path (jks, pfx)        | `null`                                                                 |
+| `es.tls.keystore.password` | Elasticsearch TLS keystore password (jks, pfx)    | `null`                                                                 |
+| `es.tls.keystore.certs`    | Elasticsearch TLS certs (only pems)               | `null`                                                                 |
+| `es.tls.keystore.keys`     | Elasticsearch TLS keys (only pems)                | `null`                                                                 |
+| `es.cluster`               | Elasticsearch cluster name                        | `elasticsearch`                                                        |
```
Comment (brasseld) [inline apim/README.md]: this property is useless, can you remove it from doc please.

**Comment by brasseld** [comment]
Please squash your commits to a single one.

**Comment by JulienFauvel** [comment]
Useless value and squash done

**Comment by brasseld** [comment]
Many thanks for the PR @JulienFauvel

**Pull request merged**